docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -67,12 +67,21 @@ Use the following example as a template for how to set the value for this proper
 ```
 components.gateway.apiml.service.additionalRegistration:
     # central API ML (in HA, for non-HA mode use only 1 hostname)
-       - discoveryServiceUrls:      https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
- 	    routes:
+       - discoveryServiceUrls: https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
+ 	     routes:
               - gatewayUrl: /
                 serviceUrl: /
 ```
 
+:::note 
+  The value `discoveryServiceUrls` should contain all instances of Discovery services for a specific API ML instance.
```

Comment:
The specific API ML instance needs bit more explanation. I was confused when reading this.

Reply:
I believe that the issue here is that there is concept of multiple domains. Where each of the domains have Zowe (Expectation is that Zowe will be setup in HA and therefore have more instances of every service)

On top of that there is a Gateway ahead of all these domains handling routing and management of all the domains. 

We need to make sure that we properly distinguish between these two pieces of software.